probes/hasBadgeNotFound/impl_test.go

```diff
@@ -0,0 +1,88 @@
+// Copyright 2023 OpenSSF Scorecard Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// nolint:stylecheck
+package hasBadgeNotFound
```

Comment:
i'm curious if we need this probe at all. If all other probes fail, it means there is no badge. We could start without this probe and see if people ask for it, maybe?

Reply:
SGTM

Reply:
Removed the probe.